### ISSUE TITLE
Mod menu still keeps saying "Update Available" despite me downloading 3 stuff that were needed for the update

### ISSUE THREAD

**Issue by Andr3MDL** [Prospector/ModMenu#714]
I'm not sure if I'm the only one receiving this type of stuff but here's my deal.
Noticed that the mod menu had an update, so I decided to get the recent one. As soon as I open the game, it still says update available.
Downloaded the other thing, and kept the recent one, still tells me to update.

![image](https://github.com/TerraformersMC/ModMenu/assets/149277730/15d0c87e-09a4-4266-b756-e6d754e9117b)
![image](https://github.com/TerraformersMC/ModMenu/assets/149277730/f358e9f4-b2ba-417b-a0eb-259d3a1469d6)

**Comment by Prospector** [comment]
Yeah, there's a bug where the beta version will try to upgrade you to the latest "release" version even though it's older than the beta.

**Comment by Nova-Katt** [comment]
Yes, I can confirm this with multiple mods.
Mod Menu obviously, and also Better Enchanted Books, and Veinminer is marked as "update available" when it points to a downgrade.
